services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/dao/ignite/IgniteConfiguration.java

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2019 The Alcor Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+        you may not use this file except in compliance with the License.
+        You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+        Unless required by applicable law or agreed to in writing, software
+        distributed under the License is distributed on an "AS IS" BASIS,
+        WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        See the License for the specific language governing permissions and
+        limitations under the License.
+*/
+
+package com.futurewei.alcor.dataplane.dao.ignite;
+
+import org.apache.ignite.Ignition;
+import com.futurewei.alcor.common.logging.Logger;
+import com.futurewei.alcor.common.logging.LoggerFactory;
+import org.apache.ignite.client.ClientException;
+import org.apache.ignite.client.IgniteClient;
+import org.apache.ignite.configuration.ClientConfiguration;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.util.Assert;
+
+import java.util.logging.Level;
+
+@Configuration
+@ComponentScan("com.futurewei.alcor.dataplane.dao")
+@EntityScan("com.futurewei.alcor.dataplane.dao")
+@ConditionalOnProperty(prefix = "ignite", name = "host")
+public class IgniteConfiguration {
```

Comment:
Most of the files in this PR could be safely removed as they are not related nor unique to DP manager, 

including but not limited to the following directories 
services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/dao/ignite
and services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/dao/repo/.